.github/workflows/ci.yml

```diff
@@ -11,8 +11,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: [3.6, 3.7, 3.8, 3.9, "3.10"]
```

Comment:
removed python 3.6, not supported
https://github.com/actions/setup-python/issues/544#issuecomment-1332535877